[2024-11-10T04:16:30.139+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-10T04:16:30.179+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Stock_data_analysis_lab2utl.check_table_stats scheduled__2024-11-08T17:00:00+00:00 [queued]>
[2024-11-10T04:16:30.192+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Stock_data_analysis_lab2utl.check_table_stats scheduled__2024-11-08T17:00:00+00:00 [queued]>
[2024-11-10T04:16:30.194+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-11-10T04:16:30.217+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): check_table_stats> on 2024-11-08 17:00:00+00:00
[2024-11-10T04:16:30.228+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Stock_data_analysis_lab2utl', 'check_table_stats', 'scheduled__2024-11-08T17:00:00+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/utl.py', '--cfg-path', '/tmp/tmpxefhzn4d']
[2024-11-10T04:16:30.231+0000] {standard_task_runner.py:91} INFO - Job 117: Subtask check_table_stats
[2024-11-10T04:16:30.233+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=439) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-10T04:16:30.236+0000] {standard_task_runner.py:63} INFO - Started process 442 to run task
[2024-11-10T04:16:30.298+0000] {task_command.py:426} INFO - Running <TaskInstance: Stock_data_analysis_lab2utl.check_table_stats scheduled__2024-11-08T17:00:00+00:00 [running]> on host 22359ceef38b
[2024-11-10T04:16:30.412+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Utkarsh Tripathi' AIRFLOW_CTX_DAG_ID='Stock_data_analysis_lab2utl' AIRFLOW_CTX_TASK_ID='check_table_stats' AIRFLOW_CTX_EXECUTION_DATE='2024-11-08T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-08T17:00:00+00:00'
[2024-11-10T04:16:30.414+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-10T04:16:30.440+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-10T04:16:30.442+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-10T04:16:31.164+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-10T04:16:31.166+0000] {logging_mixin.py:188} INFO - First row in the table: (datetime.date(2024, 7, 5), 221.65, 226.45, 221.65, 226.34, 60412408, 'AAPL')
[2024-11-10T04:16:31.166+0000] {connection.py:762} INFO - closed
[2024-11-10T04:16:31.219+0000] {connection.py:768} INFO - No async queries seem to be running, deleting session
[2024-11-10T04:16:31.283+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-10T04:16:31.285+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-10T04:16:31.296+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Stock_data_analysis_lab2utl, task_id=check_table_stats, run_id=scheduled__2024-11-08T17:00:00+00:00, execution_date=20241108T170000, start_date=20241110T041630, end_date=20241110T041631
[2024-11-10T04:16:31.338+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-10T04:16:31.357+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-10T04:16:31.363+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
